iptables-module/xt_mpeg2ts.c

```diff
@@ -271,8 +273,9 @@ struct pid_data_t {
 
 /* Data to match a stream / connection */
 struct mpeg2ts_stream_match { /* Like xt_hashlimit: dsthash_dst */
-	__be32 dst_addr; /* MC addr first */
-	__be32 src_addr;
+	int version;
```

Comment:
This structure were explicitly kept small, as a memcmp() is done on each packet, thus this change would hurt performance...